modules/core/src/main/scala/scala/build/internals/CodeWrapper.scala

```diff
@@ -40,3 +49,10 @@ object CodeWrapper {
   def mainClassObject(className: Name): Name =
     Name(className.raw ++ "_sc")
 }
+
+case class WrapperParams(
+  topWrapperLineCount: Int,
+  userCodeLineCount: Int,
+  bottomWrapperLineCount: Int,
+  userCodeNestingLevel: Int
```

Comment:
```suggestion
  userCodeNestingLevel: Int
```
Is this used?

Reply:
Actually, it's never used. Good catch, i've removed it